scripts/run.sh

```diff
@@ -3,6 +3,20 @@ set -Euo pipefail
 trap 's=$?; echo "$0: Error on line "$LINENO": $BASH_COMMAND"; exit $s' ERR
 IFS=$'\n\t'
 
+# XRD Write locations
+XRDWURL="xroots://dtn2201.jlab.org/"
+XRDWBASE=${XRDWBASE:-"/eic/eic2/EPIC"}
+
+# XRD Read locations
+XRDRURL="root://dtn-eic.jlab.org/"
+XRDRBASE="/work/eic2/EPIC"
+
+# Check xrootd connectivity
+export BEARER_TOKEN=$(cat ${_CONDOR_CREDS}/eic.use)
+if !(httokendecode -H && xrdfs ${XRDWURL} ls ${XRDWBASE} && xrdfs ${XRDRURL} ls ${XRDRBASE}); then
```

Comment:
Is the output of `httokendecode -H` something you are ok with appearing in logs?